tests/acceptance/acceptancesuite.js

```diff
@@ -180,3 +180,37 @@ test('Facets, pagination, and filters do not persist accross experience links',
   await t.expect(viewAllLink).contains('referrerPageUrl');
   await verifyCleanLink(viewAllLink);
 });
+
+fixture`Performance marks on search`
```

Comment:
I'm not sure this really fits the criteria for an acceptance test (at least how we typically use them at Yext). We're not testing a particular user flow or interaction here (i.e. I can click on an autocomplete and be redirected to another page with the query auto-populated). We're verifying that performance marks are present and accounted for. I think it'd be more appropriate to have a Jest test suite for this. The suite would have a number of different tests ensuring the various marks are populated. I think these tests would be unit tests. Possibly integration tests. I totally recognize that it's difficult to write good unit tests for the SDK. That's a consequence of a few things, including it's architecture. So, I think it's fine to keep this here for now. We definitely want the coverage. We should make an item to create a unit test suite later on (maybe after the big core refactor). The issue with acceptance tests is that they're slow. They spin up a browser and have a decent amount of overhead. More so than unit tests. It can get to a point where the acceptance test suite is so slow that nobody wants to run it locally. That's what happened with Entity Edit.  So, we want to minimize the number of acceptance tests when possible.